bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/SecurityApi.java

```diff
@@ -101,10 +101,13 @@ public void changeStatus(String localCameraURL, boolean setOn) throws NetatmoExc
         post(uriBuilder, ApiResponse.Ok.class, null, null);
     }
 
-    public void changeFloodLightMode(String localCameraURL, FloodLightMode mode) throws NetatmoException {
-        UriBuilder uriBuilder = UriBuilder.fromUri(localCameraURL).path(PATH_COMMAND).path(SUB_PATH_FLOODLIGHTSET);
+    public void changeFloodLightMode(String homeId, String cameraId, FloodLightMode mode) throws NetatmoException {
+        UriBuilder uriBuilder = getAppUriBuilder(PATH_STATE);
         uriBuilder.queryParam("config", "%7B%22mode%22:%22" + mode.toString() + "%22%7D");
```

Comment:
Is this queryParam still needed now that you send a payload with the target mode in it ?

Reply:
You are right, that query parameter isn't required anymore. It is now removed.